RELEASING.rst

```diff
@@ -14,60 +14,90 @@ Preparing: Automatic Method
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 We have developed an automated workflow for releases, that uses GitHub workflows and is triggered
-by opening an issue.
+by `manually running <https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow>`__
+the `prepare-release-pr workflow <https://github.com/pytest-dev/pytest/actions/workflows/prepare-release-pr.yml>`__
+on GitHub Actions.
 
-Bug-fix releases
-^^^^^^^^^^^^^^^^
+The automation will decide the new version number based on the following criteria:
 
-A bug-fix release is always done from a maintenance branch, so for example to release bug-fix
-``5.1.2``, open a new issue and add this comment to the body::
+- If the "major release" input is set to "yes", release a new major release
+  (e.g. 7.0.0 -> 8.0.0)
+- If there are any ``.feature.rst`` or ``.breaking.rst`` files in the
+  ``changelog`` directory, release a new minor release (e.g. 7.0.0 -> 7.1.0)
+- Otherwise, release a bugfix release (e.g. 7.0.0 -> 7.0.1)
+- If the "prerelease" input is set, append the string to the version number
+  (e.g. 7.0.0 -> 8.0.0rc1), if "major" is set, and "prerelease" is set to `rc1`)
 
-    @pytestbot please prepare release from 5.1.x
+Bug-fix and minor releases
+^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Where ``5.1.x`` is the maintenance branch for the ``5.1`` series.
+Bug-fix and minor releases are always done from a maintenance branch. First,
+consider double-checking the ``changelog`` directory to see if there are any
+breaking changes or new features.
 
-The automated workflow will publish a PR for a branch ``release-5.1.2``
-and notify it as a comment in the issue.
+For a new minor release, first create a new maintenance branch from ``main``::
 
-Minor releases
-^^^^^^^^^^^^^^
+     git fetch --all
+     git branch 7.1.x upstream/main
+     git push upstream 7.1.x
 
-1. Create a new maintenance branch from ``main``::
+Then, trigger the workflow with the following inputs:
 
-        git fetch --all
-        git branch 5.2.x upstream/main
-        git push upstream 5.2.x
+   - branch: **7.1.x**
```

Comment:
This is not rendering correctly:

![image](https://user-images.githubusercontent.com/1085180/123426201-7b489900-d599-11eb-902f-ef701ac847d8.png)

I think it needs to be dedented?